sample.env

```diff
@@ -14,3 +14,4 @@ REACT_APP_APPBAR_LOGO=iRODS-logo.jpg
 REACT_APP_LOGIN_LOGO=iRODS-logo-1.png
 REACT_APP_BRANDING_NAME=Zone Management Tool
 REACT_APP_PRIMARY_COLOR=#04bdaf
+REACT_APP_MIN_GB=10
```

Comment:
this feels... almost too high level... can we just make it a variable in the checkfile itself?  same as the 'interval_in_seconds'.   this name doesn't have meaning across the entire app.

how about... `minimum_size_in_gb`... at top of unixfilesystem-resc-space.js.

Reply:
ohhh yep, i really like that idea